dix/ut/client_ut.c

```diff
@@ -903,7 +903,7 @@ static int dix_cctg_records_del(struct m0_dix    *index,
 	m0_cas_req_init(&creq, cctg_rpc_sess(sdev_idx),
 			m0_locality0_get()->lo_grp);
 	m0_cas_req_lock(&creq);
-	rc = m0_cas_del(&creq, &cctg_id, keys, NULL, 0);
+	rc = m0_cas_del(&creq, &cctg_id, keys, NULL, COF_VERSIONED);
```

Comment:
Can we have two separate functions:
one is to use `0` and another to use `COF_VERSIONED`?

So these two cases will be covered.

Reply:
This is a Unit test.
So, if we can test:
case 1: with the flag 0 and
case 2: with the flag COF_VERSIONED
That will cover more scenarios in our code.

Reply:
I changed the dgmode UTs to run with both `0` and `COF_VERSIONED`. However, dix record operations use versioning regardless (new unconditional default), and the dgmode delete test shows some issues switching between versioned and non-versioned values (see 6260a5943f2). Not certain these UT changes are helpful

Reply:
Thank you.